Jonathan/jonathan_library_challenge.py

```diff
@@ -0,0 +1,200 @@
+    },
+    {
+        'title': 'JUST MERCY',
+        'author': 'Bryan Stevenson',
+        'description': 'A civil rights lawyer and MacArthur grant recipient’s memoir of his decades of work to free innocent people condemned to death.'
+    },
+    {
+        'title': 'BORN A CRIME',
+        'author': 'Trevor Noah',
+        'description': 'A memoir about growing up biracial in apartheid South Africa by the host of “The Daily Show.”'
+    },
+    {
+        'title': 'THE WARMTH OF OTHER SUNS',
+        'author': 'Isabel Wilkerson',
+        'description': 'An account of the Great Migration of 1915-70, in which six million African-Americans abandoned the South.'
+    },
+    {
+        'title': 'THE COLOR OF LAW',
+        'author': 'Richard Rothstein',
+        'description': 'A case for how the American government abetted racial segregation in metropolitan areas across the country.'
+    },
+    {
+        'title': 'THE NEW JIM CROW',
+        'author': 'Michelle Alexander',
+        'description': 'A law professor on the “war on drugs” and its role in the disproportionate incarceration of Black men.'
+    },
+    {
+        'title': 'THE TRUTHS WE HOLD',
+        'author': 'Kamala Harris',
+        'description': 'A memoir by the daughter of immigrants who is now a California senator and the 2020 Democratic candidate for vice president.'
+    },
+    {
+        'title': 'SAPIENS',
+        'author': 'Yuval Noah Harari',
+        'description': 'How Homo sapiens became Earth’s dominant species.'
+    },
+    {
+        'title': 'BRAIDING SWEETGRASS',
+        'author': 'Robin Wall Kimmerer',
+        'description': 'A botanist and member of the Citizen Potawatomi Nation espouses having an understanding and appreciation of plants and animals.'
+    },
+    {
+        'title': 'MY GRANDMOTHER\'S HANDS',
+        'author': 'Resmaa Menakem',
+        'description': 'A therapist who specializes in trauma, body-centered psychotherapy and violence prevention explains racism\'s effect on the body.'
+    },
+    {
+        'title': 'ON TYRANNY',
+        'author': 'Timothy Snyder',
+        'description': 'Twenty lessons from the 20th century about the course of tyranny.'
+    },
+    {
+        'title': 'THE ROAD TO UNFREEDOM',
+        'author': 'Timothy Snyder',
+        'description': 'Snyder explores Russian attempts to influence Western democracies and the influence of philosopher Ivan Ilyin on Russian President Vladimir Putin and the Russian Federation in general.'
+    }
+]
+print()
+# 2.0
+# Describe the structure of the data in books_with_details. What types of data are nested within others? How do you know?
+# Answer: these are keys and values both strings' all dictonaries 
+# 2.1
+# Create a function 'count_books' that returns the number of books in the books_with_details list
+def count_books(books_with_details):
+    print(len(books_with_details))
```

Comment:
According to the prompt, we are returing instead of printing, so you should use return instead of print.